site/content/en/docs/tasks/integrate_a_custom_job.md

```diff
@@ -0,0 +1,58 @@
+---
+title: "Integrate a custom Job"
+date: 2023-07-25
+weight: 8
+description: >
+  Integrate a custom Job with Kueue.
+---
+# Integrate a custom Job with Kueue
+
+Kueue integrates with a couple of jobs, including Kubernetes batch Job, MPIJob, RayJob or JobSet.  
+If there is another Job, which you would like supported, please consider contributing. 
+This page shows how to integrate a custom Job with Kueue.
+
+## What you need to specify
+
+1. Configuration
+2. Controller
+3. Webhook
+4. Adjust build system
+
+## Configuration
+
+Add your framework name to `.integrations.frameworks` in [controller_manager_config.yaml](https://github.com/kubernetes-sigs/kueue/blob/main/config/components/manager/controller_manager_config.yaml)
+
+## Controller
+
+Provide your code of the controller under a dedicated directory. This directory need to under the `./pkg/controller/jobs/` folder.
+
+1. Register the new framework
```

Comment:
where?

Reply:
You mean I need to mention where need to register the new framework?

Reply:
> where?

You mean I need to mention where need to register the new framework?

Reply:
yes please